### DISCUSSION TITLE
Is it possible to designate a directory to save the storage when creating a new study?

### DISCUSSION THREAD

**Discussion by seanswyi** [optuna/optuna#4629] [Q&A]
As of now I'm just doing `optuna.create_study(storage="sqlite:///{study_name}")` to create a new storage DB for my studies. However, I'd rather store them in a separate directory like `~/optuna-study-storage` or something. Is it possible to do this? If not, why? Thanks.

**Comment by nzw0301** [comment]
I suppose so. You can specify the directory in the storage URL.
